app/models/solidus_paypal_commerce_platform/order_simulator.rb

```diff
@@ -49,15 +49,17 @@ def add_line_items
     end
 
     def build_address(address)
-        country = Spree::Country.find_by(iso: address[:country_code])
+        country = ::Spree::Country.find_by(iso: address[:country_code])
```

Comment:
Just curious: did the missing colons cause any troubles?

Reply:
Not in this PR, but in #43 I add in a decorator namespaced to `SolidusPaypalCommercePlatform::Spree::AddressDecorator`, and Rails gets confused since it doesn't know if I want to acccess `Spree::Country` or `SolidusPaypalCommercePlatform::Spree::Country`. I figured it was better to start being explicit about what class we're trying to call.